ReSharper.FSharp/test/src/FSharp.Intentions.Tests/src/QuickFixes/AddSetterFIxTest.fs

```diff
@@ -0,0 +1,15 @@
+namespace JetBrains.ReSharper.Plugins.FSharp.Tests.Intentions.QuickFixes
+
+open JetBrains.ReSharper.Plugins.FSharp.Psi.Features.Daemon.QuickFixes
+open NUnit.Framework
+
+type AddSetterFixTest() =
+    inherit FSharpQuickFixTestBase<AddSetterFix>()
+    override x.RelativeTestDataPath = "features/quickFixes/addSetterFix"
+
+    [<Test>] member x.``Auto property 01``() = x.DoNamedTest()
+    [<Test>] member x.``Auto property 02 - Static``() = x.DoNamedTest()
+    [<Test>] member x.``Auto property 03 - With getter``() = x.DoNamedTest()
+    [<Test>] member x.``Auto property 04 - Abstract``() = x.DoNamedTest()
+    [<Test; NotAvailable>] member x.``Not available 01 - Virtual``() = x.DoNamedTest()
+    [<Test; NotAvailable>] member x.``Not available 02 - Signature``() = x.DoNamedTestWithSignature()
```

Comment:
Maybe we could support updating the signatures as well?
We could check that all declarations can be updated instead of trying to get a single one and then checking it.

Reply:
First, we need to fix the accessor tree for IMemberSignature

Reply:
We can merge it for existing cases first and cover more cases in subsequent PRs if you want. It'd be easier to test it earlier and prevent the PR from staling. What do you think?